### PR TITLE
Automated OCP-27654

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review. Thx!

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/notification.xyaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+click_notification_drawer:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/notification.xyaml]: This can be used with `click_masthead_button`

Comment (yapei) [reply lib/rules/web/admin_console/4.4/notification.xyaml]: I would recommend create a new file `masthead.xyaml` with all masthead related rules into it

File: lib/rules/web/admin_console/4.4/notification.xyaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  elements:
+  - selector:
+      xpath: //button[contains(@aria-label,'Notification Drawer')]
+    op: click
+    timeout: 30
+open_notification_drawer:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/notification.xyaml]: Normally we'd better not mix click & check operation in one action. Let me try to explain
Given these scenarios in feature 
```
    When I run the :open_notification_drawer web action
    Then the step should succeed
    When I run the :close_notification_drawer web action
    Then the step should succeed
```
At the first glance, we just notice that we're opening & close notification drawer, didn't know we also checked that `Notification` headings appear & disappear

Comment (yapei) [reply lib/rules/web/admin_console/4.4/notification.xyaml]: This time we can just leave them as they are

**Comment by SSshahan** [comment]
@pruan-rht please help to merge, thanks

**Pull request merged**